### PR TITLE
ci: use the RHEL 8.5 repos on the mirror

### PR DESCRIPTION
**Pull request by miabbott**
We want to test RHCOS using RHEL 8.5 until we can switch to using RHEL
8.6, so use the newly created 8.5 repo.

See: https://github.com/openshift/release/pull/26193

### CODE REVIEW

File: ci/prow-build-test-qemu.sh
Comment (cgwalters) [inline ci/prow-build-test-qemu.sh:35]: This is fine as is, but it's another place to bump the RHEL version.  Let's figure out how to dedup all this after recent PRs merge though, don't block here!

**Review by cgwalters — APPROVED** [review]

File: ci/prow-build-test-qemu.sh
Comment (miabbott) [reply ci/prow-build-test-qemu.sh:35]: Ack.  Will do!

**Comment by miabbott** [comment]
`Updating rpm-md repo 'rhel-8-advanced-virt': cannot update repo 'rhel-8-advanced-virt': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried; Last error: Status code: 404 for http://base-4-11-rhel85.ocp.svc.cluster.local/rhel-8-advanced-virt/repodata/repomd.xml`

I think the repo definition needs a tweak, will chase that down

**Comment by miabbott** [comment]
> I think the repo definition needs a tweak, will chase that down

https://github.com/openshift/release/pull/26305

**Comment by miabbott** [comment]
Messed up the baseurl for RHEL AV :disappointed:  - https://github.com/openshift/release/pull/26334

**Comment by miabbott** [comment]
CI failed on the `iso-install` branch:

```
+ kola testiso -S --scenarios pxe-install,pxe-offline-install,iso-install,iso-offline-install,iso-live-login,iso-as-disk --output-dir tmp/kola-metal
Testing scenarios: [pxe-install pxe-offline-install iso-install iso-offline-install iso-live-login iso-as-disk]
Ignoring verification of signature on metal image
Could not open option rom '/usr/share/qemu/pxe-rtl8139.rom': No such file or directory
Successfully tested scenario pxe-install for 411.84.202202221344-0 on bios (metal)
Could not open option rom '/usr/share/qemu/pxe-rtl8139.rom': No such file or directory
Successfully tested scenario pxe-offline-install for 411.84.202202221344-0 on bios (metal)
Error: scenario iso-install: entered emergency.target in initramfs
2022-02-22T15:47:28Z cli: scenario iso-install: entered emergency.target in initramfs
```

But the failed artifacts didn't get saved, so not sure what happened.  Will try to reproduce locally.

**Comment by miabbott** [comment]
Didn't reproduce locally

/retest

**Review by jmarrero — APPROVED** [review]

**Comment by miabbott** [comment]
/hold

Need to get a `cosa` pod on `build02` to investigate why the `iso-install` is failing

**Comment by bgilbert** [comment]
Yeah, it looks consistent with #666.  I'll reopen that bug and adding some observations there.

**Review by ravanelli — APPROVED** [review]
LGTM

**Review by saqibali-2k — APPROVED** [review]
/lgtm

**Pull request merged**
